packages/SwingSet/misc-tools/store-stats/src/store-stats.js

```diff
@@ -0,0 +1,160 @@
+import { StatsD } from 'hot-shots';
+
+const dogstatsd = new StatsD({
+  errorHandler(error) {
+    console.log('StatsD socket errors: ', error);
+  },
+});
+
+/**
+ * @param {import('better-sqlite3').Database} db
+ */
+export const makeSwingstore = db => {
+  const dbTool = () => {
+    const prepare = (strings, ...params) => {
+      const dml = strings.join('?');
+      return { stmt: db.prepare(dml), params };
+    };
+    const sql = (strings, ...args) => {
+      const { stmt, params } = prepare(strings, ...args);
+      return stmt.all(...params);
+    };
+    sql.get = (strings, ...args) => {
+      const { stmt, params } = prepare(strings, ...args);
+      return stmt.get(...params);
+    };
+    sql.iterate = (strings, ...args) => {
+      const { stmt, params } = prepare(strings, ...args);
+      return stmt.iterate(...params);
+    };
+    return sql;
+  };
+
+  const sql = dbTool();
+
+  /** @param {string} key */
+  const kvGet = key => sql.get`select * from kvStore where key = ${key}`.value;
+
+  const kvAll = key => sql.iterate`select * from kvStore where key GLOB ${key}`;
+
+  return Object.freeze({
```

Comment:
Let's consider using `harden` here.. if we're going to freeze it, we might as well go all the way. It's not so important for a CLI tool like this (this returned API object is not being exposed to any malicious/untrusted code), but it's not a bad habit to get into.

`return harden({ ..`, and then the `index.js` needs to start with `import '@endo/init'`, and we need to add `@endo/init` to the package.json. That's the pattern we use for our "Hardened JS" style. We usually harden everything exported by a module (`export const fname = harden(() => { ..`), as well as any object exposed to the importers of the module.